src/app/components/home/home.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, OnInit, ViewChild } from '@angular/core';
 import { Pet, PetStatus } from '../../model';
 import { PetsService } from '../../service';
 import { PetDescriptionComponent } from '../pet-description/pet-description.component';
+// import { OverlayService } from '../../service/overlay/overlay.service';
```

Comment:
Changes in this file are not about Overlay Service or Sign In Component. Please try to avoid redundant changes in pull requests

Reply:
I forgot to delete the unmentioned code in this  file